packages/vue/src/stories/releases/v0.9.0.stories.mdx

```diff
@@ -54,12 +56,12 @@ Then run `yarn` or `npm i` to re-install your local dependencies.
 
 | Before                                                                      | v0.6.4                           | v0.7.0                  | v0.9.0                    |
 | --------------------------------------------------------------------------- | -------------------------------- | ----------------------- | ------------------------- |
-| \$font-size-extra-small-mobile: 0.625rem                                    | undefined                        | --font-2xs: 0.625rem    | undefined                 |
```

Comment:
you can write `div` HTML element here, so maybe try to make a table with CSS and HTML instead of markdown? :)

Reply:
html tables added instead of markdown :)